frontend/src/pages/LogsPage/useGetLogs.ts

```diff
@@ -37,103 +44,105 @@ export const useGetLogs = ({
 	//
 	// If the user scrolls forward to get the next 100 logs, the server will say that hasPreviousPage is `true` since we're on page 2.
 	// Hence, we track the initial information (where "window" is effectively multiple pages) to ensure we aren't making requests unnecessarily.
-	const [windowInfo, setWindowInfo] = useState<PageInfo>({
-		hasNextPage: false,
-		hasPreviousPage: false,
-		startCursor: '',
-		endCursor: '',
-	})
+	const [windowInfo, setWindowInfo] = useState<PageInfo>(initialWindowInfo)
 	const [loadingAfter, setLoadingAfter] = useState(false)
 	const [loadingBefore, setLoadingBefore] = useState(false)
 	const queryTerms = parseLogsQuery(query)
 	const serverQuery = buildLogsQueryForServer(queryTerms)
 
-	const [getLogs, { data, loading, error, refetch, fetchMore }] =
-		useGetLogsLazyQuery({
-			variables: {
-				project_id: project_id!,
-				at: logCursor,
-				direction: Types.LogDirection.Desc,
-				params: {
-					query: serverQuery,
-					date_range: {
-						start_date: moment(startDate).format(LOG_TIME_FORMAT),
-						end_date: moment(endDate).format(LOG_TIME_FORMAT),
-					},
+	useEffect(() => {
+		setWindowInfo(initialWindowInfo)
+	}, [query])
```

Comment:
should this also reset when the `startDate` and `endDate` change, since those drive the initial `GetLogs` query?

https://www.loom.com/share/3b9c1aabc3ef497e8ef2937a257ed53c